pkg/informer/informer.go

```diff
@@ -425,14 +545,14 @@ func withGVRPartialMetadata(scope apiextensionsv1.ResourceScope, kind, singular
 	}
 }
 
-type gvrPartialMetadata struct {
+type GVRPartialMetadata struct {
```

Comment:
Why does this have to leak upwards?

Reply:
Well, it wold be used by external implementations of `GVRSource`